src/app/lib/config/services/news.js

```diff
@@ -14,6 +14,39 @@ const news = {
   themeColor: `${C_POSTBOX}`,
   twitterCreator: '@BBCNews',
   twitterSite: '@BBCNews',
+  translations: {
+    error: {
+      404: {
+        statusCode: '404',
+        title: 'Page can not be found',
```

Comment:
```suggestion
        title: 'Page cannot be found',
```

This doesn't match the UX specification, & leaves the title reading rather oddly.